src/renderer/src/components/Select.tsx

```diff
@@ -16,6 +16,8 @@ type TProps = {
   disabled?: boolean
   onSelect?: (value: string) => void
   children: React.ReactNode
+  bgColor?: string
```

Comment:
It would be better to receive a className and use StyleMerge on Select.Root. This makes this component more customizable;